pvnet_summation/data/datamodule.py

```diff
@@ -164,7 +163,7 @@ def _get_premade_batches_datapipe(self, subdir, shuffle=False, add_filename=Fals
         file_pipeline = FileLister(f"{self.batch_dir}/{subdir}", masks="*.pt", recursive=False)
 
         if shuffle:
-            file_pipeline = file_pipeline.shuffle(buffer_size=1000)
+            file_pipeline = file_pipeline.shuffle(buffer_size=10_000)
```

Comment:
No particular reason. Just that at this stage all we'll being doing is loading up to 1000 or 10,000 file names before shuffling them. I thought that since the filenames don't take up any amount of space why not use more and have a more complete shuffle.